webapp/src/rudderstack.ts

```diff
@@ -0,0 +1,11 @@
+import * as rudderanalytics from 'rudder-sdk-js';
+rudderanalytics.load(
+    '287DKMJUsG92u2S588OJ5CAisGd',
```

Comment:
@stephaniebeagle do you know if this key is important and we should not expose it in the Front end?

Reply:
It's a write key Rudderstack generated for us to specify which app the events are coming from. I don't think it's sensitive